docs/src/tutorials/applications/optimal_power_flow.jl

```diff
@@ -564,12 +536,3 @@ DataFrames.DataFrame(;
 
 # **MATPOWER data format**:
 # [MATPOWER manual](https://matpower.org/docs/MATPOWER-manual.pdf); see especially Appendix B "Data File Format" and Table B-3.
-
-# **PowerModels.jl**:
-# the Julia/JuMP package [PowerModels.jl](https://lanl-ansi.github.io/PowerModels.jl/stable/) provides
-# an open-source framework for a broad range of power flow formulations along with utilities for working with detailed network data.
-
-# **Ipopt solver**:
-# Wächter, A., Biegler, L.
-# [_On the implementation of an interior-point filter line-search algorithm for large-scale nonlinear programming._](https://doi.org/10.1007/s10107-004-0559-y)
-# Math. Program. 106, 25—57 (2006).
```

Comment:
I removed PowerModels because we already link to it above. And I removed the paper reference to Ipopt because I don't know if it's relevant for the tutorial? Users don't need to read the paper to use Ipopt.

Reply:
That's fine. I guess we would be referencing every solver paper if in every tutorial otherwise.

Reply:
There's an argument for "you should cite the paper if you use Ipopt," but that's a different discussion.

Reply:
Yes, that was in my mind when I added it. But it's a broader discussion for the whole docs.